### PR TITLE
build: CLI container to use in webstudio builder publish process

### PR DESCRIPTION
**Pull request by istarkov**
## Description

Deployed tags:

image

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+node_modules
+.pnp
+.pnp.js
+yarn.lock
```
Comment (TrySound) [inline .dockerignore:7]: kinda not relevant

File: packages/cli/templates/cloudflare/vite.config.ts [hunk 2]
```diff
@@ -5,6 +5,10 @@ import {
 import { defineConfig } from "vite";
 import tsconfigPaths from "vite-tsconfig-paths";
 
-export default defineConfig({
-  plugins: [remixCloudflareDevProxy(), remix(), tsconfigPaths()],
-});
+export default defineConfig(({ mode }) => ({
+  plugins: [
+    mode === "production" ? undefined : remixCloudflareDevProxy(),
```
Comment (TrySound) [inline packages/cli/templates/cloudflare/vite.config.ts:10]: remix doesn't handle it under the hood?

File: devops/cli/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+RUN apk update && apk add curl rclone postgresql14-client --no-cache
+
+COPY ./ /cli
+WORKDIR /cli
+
+ENV PNPM_HOME="/.pnpm/store"
+ENV PATH="$PNPM_HOME:$PATH"
+
+RUN corepack enable pnpm && mkdir -p $PNPM_HOME
+RUN pnpm i -g zx
```
Comment (TrySound) [inline devops/cli/Dockerfile:21]: why zx?

Comment (istarkov) [reply devops/cli/Dockerfile:21]: easy to use in saas (and already used there)

File: packages/cli/templates/cloudflare/vite.config.ts (hunk 2 above)
Comment (istarkov) [reply packages/cli/templates/cloudflare/vite.config.ts:11]: Added comment, bug in remix proxy
see `//without this, remixCloudflareDevProxy trying to load workerd even for production (it's not needed for production)`

**Pull request merged**